packages/google-pay-integration/e2e/GooglePayBraintreeInPaymentStep.spec.ts

```diff
@@ -51,6 +53,20 @@ test.describe('Google Pay Braintree', () => {
             void route.fulfill({ ...responseProps, body: payment });
         });
 
+        await page.route(
+            /.*\/api\/storefront\/checkout-settings\?checkoutId=.*/,
+            async (route, request) => {
+                if (request.url().includes(`checkoutId=${mockCartId}`)) {
+                    return route.fulfill({
+                        ...responseProps,
+                        body: JSON.stringify(checkoutSettings),
+                    });
+                }
+
+                await route.fallback();
+            },
+        );
```

Comment:
We should avoid mocking `checkout-settings` because if the structure of `checkout-settings` changes, we can convert all old recordings into new format here.
https://github.com/bigcommerce/checkout-js/blob/25633d5cc3cae0f800cbfdd101fca9487c70e235/packages/test-framework/src/fixture/pageObject/playwright/PollyObject.ts#L121
A mocking response like this will just break the test in the scenario above.

Reply:
Thanks! I got it. I will think about the different way to solve it